src/biogeophys/LunaMod.F90

```diff
@@ -897,11 +903,11 @@ subroutine NitrogenAllocation(FNCa,forc_pbot10, relh10, CO2a10,O2a10, PARi10,PAR
   tleafd10c = min(max(tleafd10, Trange1), Trange2)    !constrain the physiological range
   tleafn10c = min(max(tleafn10, Trange1), Trange2)    !constrain the physiological range
   ci = 0.7_r8 * CO2a10 
-  JmaxCoef = Jmaxb1 * ((hourpd / 12.0_r8)**2.0_r8) * (1.0_r8 - exp(-relhExp * max(relh10 - minrelh, 0.0_r8) / &
+  JmaxCoef = Jmaxb1 * dayl_factor * (1.0_r8 - exp(-relhExp * max(relh10 - minrelh, 0.0_r8) / &
        (1.0_r8 - minrelh)))
   do while (PNlcoldi .NE. PNlc .and. jj < 100)      
-     Fc = VcmxTKattge(tair10, tleafd10c) * Fc25
-     Fj = JmxTKattge(tair10, tleafd10c) * Fj25
+     Fc = VcmxTLeuning(tair10, tleafd10c) * Fc25
```

Comment:
Another thing to talk about on Wednesday

Reply:
I am hesitant about switching to Leuning. From my recollection, this essentially turns off the photosynthetic temperature acclimation functions. I understand that the Kattge acclimation representation is problematic in high latitudes due to the 11-degree C minimum temperature threshold, but switching this globally will also have important implications in other regions. Acclimation is important for tropical vegetation starting in 1850, for example.

I know that Leah showed its importance for high latitude vegetation, but we need to think more carefully about the global impacts before switching. Ultimately, it might be nice to test out Nick Smith's optimized photosynthesis code, as it may resolve some of these problems.

Reply:
I also agree that removing acclimation from lower latitudes is probably not ideal. The choice to use Leuning for Arctic PFTs might be something to consider. In any case, the Kattge acclimation biases Jmax and Vcmax high below 11C, which is a good bit of the Arctic-Boreal, especially in spring. Using Leuning instead, decreases the high bias of the needleleaf evergreen trees in particular, which typically try to begin photosynthesis very early in spring.

Reply:
Eventually, maybe we want this to be a namelist switch in the code, but for initial  testing we'll stick with Kattge @ekluzek